wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,415 @@
+| **Grup C** | ::{ flag=ID }:: [\[KonG-69\]](https://osu.ppy.sh/users/8847412) (14 Point), ::{ flag=ID }:: [rizki20](https://osu.ppy.sh/users/6523286) (12 Point), ::{ flag=ID }:: [-MyLoveIka-](https://osu.ppy.sh/users/12039798) (6 Point), ::{ flag=ID }:: [Vels](https://osu.ppy.sh/users/5319494) (3 Point), ::{ flag=ID }:: [ElinLYPK](https://osu.ppy.sh/users/5198950) (1 Point) | [#1](https://osu.ppy.sh/community/matches/52985075) |
+| **Grup D** | ::{ flag=ID }:: [StyxFearNite](https://osu.ppy.sh/users/8390933) (15 Point), ::{ flag=ID }:: [budiex112](https://osu.ppy.sh/users/8867693) (14 Point), ::{ flag=ID }:: [agung0210](https://osu.ppy.sh/users/4663559) (7 Point), ::{ flag=ID }:: [Kawaisou-](https://osu.ppy.sh/users/11248741) (0 Point), ::{ flag=ID }:: [Ihram-San](https://osu.ppy.sh/users/9908055) (0 Point) | [#1](https://osu.ppy.sh/community/matches/52987435) |
+
+| Sunday, 30 June 2019 |  |  |
+| :-: | :-- | :-: |
+| **Grup E** | ::{ flag=ID }:: [MATEH ON](https://osu.ppy.sh/users/8136810) (13 Point), ::{ flag=ID }:: [Noch Einen](https://osu.ppy.sh/users/12668735) (13 Point), ::{ flag=ID }:: [Yuusha006](https://osu.ppy.sh/users/11214161) (10 Point), ::{ flag=ID }:: [THEmpire](https://osu.ppy.sh/users/12142847) (0 Point), ::{ flag=ID }:: [PetrikPintar](https://osu.ppy.sh/users/13489565) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53010360) |
+| **Grup F** | ::{ flag=ID }:: [Merlinch](https://osu.ppy.sh/users/4901971) (13 Point), ::{ flag=ID }:: [Its A Name](https://osu.ppy.sh/users/8724363) (11 Point), ::{ flag=ID }:: [Kurami](https://osu.ppy.sh/users/10184990) (10 Point), ::{ flag=ID }:: [Felter](https://osu.ppy.sh/users/5865319) (2 Point), ::{ flag=ID }:: [Fathan2Gaming](https://osu.ppy.sh/users/11982790) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53006427) |
+| **Grup G** | ::{ flag=ID }:: [alvinrainbow](https://osu.ppy.sh/users/4783042) (15 Point), ::{ flag=ID }:: [anzutto232](https://osu.ppy.sh/users/13408635) (15 Point), ::{ flag=ID }:: [pakyan29](https://osu.ppy.sh/users/7879632) (6 Point), ::{ flag=ID }:: [ilhamsyah849](https://osu.ppy.sh/users/4679215) (0 Point), ::{ flag=ID }:: [\[AST\]Rosary](https://osu.ppy.sh/users/12504843) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53005726) |
+| **Grup H** | ::{ flag=ID }:: [\[-TiaZark-\]](https://osu.ppy.sh/users/14294140) (13 Point), ::{ flag=ID }:: [Dil-](https://osu.ppy.sh/users/3796282) (11 Point), ::{ flag=ID }:: [ReZ\_TB](https://osu.ppy.sh/users/9113955) (6 Point), ::{ flag=ID }:: [Hilmi Ghazian](https://osu.ppy.sh/users/3200342) (6 Point), ::{ flag=ID }:: [RizSama](https://osu.ppy.sh/users/9281315) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53009213) |
+
+## General Rules
+
+- There is no limit to the amount of participants.
+- Participants should register on the links that are already provided.
```

Comment:
what links

Reply:
links from the beginning of the article

Reply:
through challonge i assume?

Reply:
it's from the forum discussion thread apparently (discussion thread is also linked in the Links section)

Reply:
i see

```suggestion
- Participants should register through the links provided in the forum thread.
```